examples/distributing/uniffi-rust-to-python-library/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "uniffi-rust-to-python-library"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+uniffi = {version = "*", features = [ "cli" ]}
+uniffi_macros = "*"
```

Comment:
These all seem odd - you probably want to do what the other examples do, and I don't think this example needs a direct dependency on `uniffi_macros` or on `uniffi_bindgen`